test/PolyhedralGeometry/polyhedral_fan.jl

```diff
@@ -119,6 +119,36 @@
   end
 end
 
+@testset "minimal_supercone" begin
```

Comment:
The following tests should be moved to the toric tests, as they test toric stuff. (Or you accept a polyhedral fan as first argument.)

Reply:
Moved to PolyhedralGeometry